_events/2021-hackathon.md

```diff
@@ -0,0 +1,44 @@
+---
+title: DTS Hackathon
+---
+
+The first DTS hackathon is organised by the DTS community and generously funded by the [dhCenter UNIL-EPFL](https://dhcenter-unil-epfl.com/) and the [École Nationale des Chartes](http://www.chartes.psl.eu/). The **goal** of this hackathon is to raise awareness about the [DTS specifications](https://w3id.org/dts), encourage their use and, ultimately, to improve the ecosystem of datasets and tools around DTS.
+
+## Format
+
+The hackathon will take place fully online between **27 September - 8 October 2021**.
+
+We propose two types of hacks:
+1. **data hacks**, aimed at exposing new or already existing textual resources via a DTS-compliant API with the goal of enlarging the variety of domains, languages and periods covered by DTS. Participants proposing a data hack *may* be affiliated with the institution owning the data, but this is not strictly necessary (e.g. in case of open data).
+2. **tool hacks**, aimed at enlarging the offer of DTS-compatible tools by extending existing software or developing new ones. Ideas for tool hacks include:
+- development of DTS client libraries in any programming language (e.g., Go, Java, Ruby);
```

Comment:
Should we add Javascript and Python, that are quite broadly used ? I feel like it should be there :)